<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>542</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>542</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>542</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Puuzel 15</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>hand.png</normaloff>hand.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 85, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>                           Computer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn1_unvisivable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_computer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn2_unvisivable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="how_win">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);</string>
      </property>
      <property name="text">
       <string>               Who won?</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>                              player</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn3_unvisivable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_you">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btn4_unvisivable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="scoreboard">
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>     You: 0       Ties: 0          Computer: 0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btn_paper">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>paper.png</normaloff>paper.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>112</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btn_scissors">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>scissors.png</normaloff>scissors.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="btn_rock">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>33</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>rock.png</normaloff>rock.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>112</width>
        <height>112</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
